File: Praktikum Pemrograman I Modul 2/Praktikum I Modul2 10-10-2023/Python/PRAK205-2310817110015-AKHMAD_CHAIDAR_ANANDA.py
import math

tinggi, sisi_miring = map(int, input().split())

alas = math.sqrt((sisi_miring*sisi_miring) - (tinggi*tinggi))
keliling = alas + tinggi + sisi_miring
luas = 0.5 * alas * tinggi

print("Alas     =",format(alas,'.0f'),'cm')
print("Tinggi   =",format(tinggi,'.0f'),'cm')
print("Keliling =",format(keliling,'.0f'),'cm')
print("luas     =",format(luas,'.0f'),'cm^2')

File: Praktikum Pemrograman I Modul 2/Praktikum I Modul2 10-10-2023/C/PRAK205-2310817110015-AKHMAD_CHAIDAR_ANANDA.c
#include<stdio.h>
#include<math.h>

int main(){

    // Deklarasi variabel
    // Alas (C), Tinggi(A), Sisi miring(B)
    int alas, tinggi, sisi_miring, keliling, luas;

    // Input
    scanf("%d %d", &tinggi, &sisi_miring);

    // Alas
    alas = sqrt((sisi_miring*sisi_miring) - (tinggi*tinggi));
    // Keliling
    keliling = alas + tinggi + sisi_miring;
    // Luas
    luas = 0.5 * alas * tinggi;

    // Output
    printf("Alas     = %d cm\n", alas);
    printf("Tinggi   = %d cm\n", tinggi);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas     = %d cm^2\n", luas);
    
    return 0;
}